Source/Shaders/GlobeFS.glsl

```diff
@@ -61,6 +61,8 @@ varying vec3 v_positionEC;
 varying vec3 v_textureCoordinates;
 varying vec3 v_normalMC;
 varying vec3 v_normalEC;
+varying float v_height;
+varying float v_slope;
```

Comment:
Surround these with the `APPLY_MATERIAL` define.

Reply:
Fixed